schemes/scheme-model.js

```diff
@@ -0,0 +1,48 @@
+//write code here
+
+const db = require('../data/dbConfig');
+
+function find () {
+    return db('schemes')
+}
+
+function findById (id) {
+    return db('schemes').where({ id }).first()
+}
+
+function findSteps(schemeId) {
+    return db('steps')
+      .join('schemes', 'schemes.id', 'scheme_id')
+      .select('steps.id', 'schemes.scheme_name', 'steps.step_number', 'steps.instructions')
+      .where('scheme_id', schemeId);
```

Comment:
Try ordering the steps by step_id here, that way all the steps will be in order thus making it easier for the user.